src/session/cookie-store/index.ts

```diff
@@ -18,54 +18,65 @@ export default class CookieSessionStore implements ISessionStore {
    * @param req HTTP request
    */
   async read(req: IncomingMessage): Promise<ISession | null> {
-    const { cookieSecret, cookieName } = this.settings;
+    try {
+      const { cookieSecret, cookieName } = this.settings;
 
-    const cookies = parseCookies(req);
-    const cookie = cookies[cookieName];
-    if (!cookie || cookie.length === 0) {
-      return null;
-    }
+      const cookies = parseCookies(req);
+      const cookie = cookies[cookieName];
+      if (!cookie || cookie.length === 0) {
+        return null;
+      }
+
+      const unsealed = await Iron.unseal(cookies[cookieName], cookieSecret, Iron.defaults);
+      if (!unsealed) {
+        return null;
+      }
 
-    const unsealed = await Iron.unseal(cookies[cookieName], cookieSecret, Iron.defaults);
-    if (!unsealed) {
+      return unsealed as ISession;
+    } catch (error) {
+      console.error(error);
       return null;
     }
-
-    return unsealed as ISession;
   }
 
   /**
    * Write the session to the cookie.
    * @param req HTTP request
    */
   async save(req: IncomingMessage, res: ServerResponse, session: ISession): Promise<void> {
-    const {
-      cookieSecret, cookieName, cookiePath, cookieLifetime
-    } = this.settings;
+    try {
+      const {
+        cookieSecret, cookieName, cookiePath, cookieLifetime
+      } = this.settings;
 
-    const {
-      idToken, accessToken, refreshToken, user, createdAt
-    } = session;
-    const persistedSession = new Session(user, createdAt);
+      const {
+        idToken, accessToken, refreshToken, user, createdAt
+      } = session;
 
-    if (this.settings.storeIdToken && idToken) {
-      persistedSession.idToken = idToken;
-    }
+      const persistedSession = new Session(user, createdAt);
 
-    if (this.settings.storeAccessToken && accessToken) {
-      persistedSession.accessToken = accessToken;
-    }
+      if (this.settings.storeIdToken && idToken) {
+        persistedSession.idToken = idToken;
+      }
```

Comment:
Any reason this changed from `access` to `id`?  I'd like to leave that stuff as alone as possible since it's not just for our use case.

Reply:
That's just github, if you look beneath on line 62, the `access` one is there. Nothing was changed here, dont know why it showed like that